src/chat_command_handler.hpp

```diff
@@ -113,6 +114,8 @@ class chat_command_handler : public map_command_handler<chat_command_handler>
 			_("Display version information."));
 		register_command("info", &chat_command_handler::do_info,
 			_("Request information about a nickname."), _("<nickname>"));
+		register_command("clear", &chat_command_handler::do_clear,
+			_("Clear messages."));
```

Comment:
I'd prefer to have "Clear chat history.", because that string is already translated; it's the documentation for the `:clear` command.